test/Serialization/Fixtures/SerializableObjectUsingTrait.php

```diff
@@ -2,7 +2,7 @@
 
 namespace BroadwaySerialization\Test\Serialization\Fixtures;
 
-use Broadway\Serializer\SerializableInterface;
+use Broadway\Serializer\Serializable as SerializableInterface;
```

Comment:
Hmm, this isn't good for UX ;) I think the trait `Serializable` has to be renamed instead. What do you think? I think we should release a new major version anyway.